src/test/scala/org/hammerlab/guacamole/reads/MDTagUtilsSuite.scala

```diff
@@ -56,8 +71,15 @@ class MDTagUtilsSuite extends FunSuite with Matchers {
 
   test("rebuild reference from multiple reads with multiple mismatches") {
     val originalReference = "AAATTGATACTCGAACGA"
+
+    /*
+        ref: AAATTGATACTCGAACGA
+         r1: AAATTGATAC
+         r2:      GCTACTCAAA
+         r3:         ACTCGAACGA
+     */
     val firstRead = TestUtil.makeRead(originalReference.substring(0, 10), "10M", "10", start = 0)
-    val secondRead = TestUtil.makeRead("GCGGGTACTCGAA", "10M", "1C5G2", start = 5)
+    val secondRead = TestUtil.makeRead("GCGGGTACTCAAA", "10M", "1C5A2", start = 5)
```

Comment:
This is actually my mistake the first time, the MDTag should be 1A5G2.  It should say what the reference bases are.